ulog/context.go

```diff
@@ -74,8 +74,7 @@ func (t trace) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 	if !j.IsValid() {
 		return fmt.Errorf("invalid span: %v", j.SpanID())
 	}
-	enc.AddUint64("span", j.SpanID())
-	enc.AddUint64("trace", j.TraceID())
-	enc.AddUint64("parent", j.ParentID())
```

Comment:
Understood, the main reason for this change is jaeger uses the hex ids in their UI and other places. We want that experience to be consistent for users.